parts/k8s/health-monitor.sh

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env bash
```

Comment:
Can we add a URL reference to the canonical source for this?

Reply:
Added below the shebang line.